src/tests/ftest/util/server_utils.py

```diff
@@ -312,6 +327,33 @@ def __init__(self, server_command, manager="Orterun", dmg_cfg=None):
         # to access the doas_servers when they are started
         self.dmg = DmgCommand(self.manager.job.command_path, dmg_cfg)
 
+    def get_params(self, test):
+        """Get values for all of the command params from the yaml file.
+
+        Use the yaml file parameter values to assign the server command and
+        orterun command parameters.
+
+        Args:
+            test (Test): avocado Test object
+        """
+        super(DaosServerManager, self).get_params(test)
+        # Get the values for the dmg parameters
+        self.dmg.get_params(test)
+
+    def get_interface_envs(self, index=0):
+        """Get the environment variable names and values for the interfaces.
+
+        Args:
+            index (int, optional): server index from which to obtain the
+                environment variable values. Defaults to 0.
+
+        Returns:
+            EnvironmentVariables: a dictionary of environment variable names
+                and their values extracted from the daos_server yaml
+                configuration file.
+
+        """
+        return self.manager.job.get_interface_envs(index)
```

Comment:
Why is this being added?  It looks like it came from an outdated copy of the file.